ironfish/src/workerPool/job.test.slow.ts

```diff
@@ -21,20 +20,4 @@ describe('Worker Pool', () => {
 
     expect(workerPool.completed).toBe(1)
   })
-
-  it('verify', async () => {
-    const { workerPool } = nodeTest
```

Comment:
Delete this test bc its loading data from wallet db for genesis block, which has old data format. Need a db migration before adding it back.